markdown_help.go

```diff
@@ -0,0 +1,161 @@
+package baker
```

Comment:
at first glance I'd had called these files `help_text` and `help_markdown`, to visually group all together. It it a convention to call them with the trailing `_help`?

Reply:
good point

Reply:
done